### ISSUE TITLE
Remove twisted.pb from the TwistedCore package

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11172]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1666|
|Type|task|
|Created|2006-04-24 01:16:55Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1666 1666
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1145841415000000 1145841415000000
changetime__1145841518000000 1145841518000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16661]) Remove flow and pair from subproject lists; add them and pb to the core blacklist

Author: exarkun
Reviewers: itamarst, radix
Fixes [#11170](https://github.com/twisted/twisted/issues/11170) and [#11172](https://github.com/twisted/twisted/issues/11172)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
